docs/en/connector-v2/sink/Snowflake.md

```diff
@@ -0,0 +1,57 @@
+# Snowflake
+
+> Snowflake sink connector
+>
+> ## Description
```

Comment:
![image](https://user-images.githubusercontent.com/45089228/231667471-7529b388-f73d-4580-8748-966a1083f1dd.png)
It looks like this part of the document needs to be formatted.